command/server/config.go

```diff
@@ -366,15 +368,28 @@ func ParseConfig(d string, logger log.Logger) (*Config, error) {
 		return nil, err
 	}
 
-	if o := list.Filter("backend"); len(o.Items) > 0 {
-		if err := parseBackends(&result, o); err != nil {
-			return nil, fmt.Errorf("error parsing 'backend': %s", err)
+	// Look for storage but still support old backend
+	if o := list.Filter("storage"); len(o.Items) > 0 {
+		if err := parseStorage(&result, o, "storage"); err != nil {
+			return nil, fmt.Errorf("error parsing 'storage': %s", err)
+		}
+	} else {
+		if o := list.Filter("backend"); len(o.Items) > 0 {
+			if err := parseStorage(&result, o, "backend"); err != nil {
+				return nil, fmt.Errorf("error parsing 'backend': %s", err)
```

Comment:
@sethvargo I thought about that but we don't really have that plumbed through right now. Realistically we will never actually remove this so over time people checking docs will just realize that it's called `storage` now and new users will always call it `storage`. This might be one of those "when I'm bored" things, to allow returning a warning instead of just an error, or allow sending the output stream into the config parsing function, to print an error.